File: auto-docs/executables/node/error_bar_bar.js
var plotly = require('plotly')('theengineear', 'o9zlr0hy6z')

var trace1 = {
  x: ["Trial 1", "Trial 2", "Trial 3"], 
  y: [3, 6, 4], 
  name: "Control", 
  error_y: {
    array: [1, 0.5, 1.5], 
    type: "data", 
    visible: true
  }, 
  type: "bar"
};
var trace2 = {
  x: ["Trial 1", "Trial 2", "Trial 3"], 
  y: [4, 7, 3], 
  name: "Experimental", 
  error_y: {
    array: [0.5, 1, 2], 
    type: "data", 
    visible: true
  }, 
  type: "bar"
};
var data = [trace1, trace2];
var layout = {barmode: "group"};

var graph_options = {filename: "error-bar-bar", fileopt: "overwrite", layout: layout, auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/python/error_bar_bar.py
# Learn about API authentication here: plot.ly/python/getting-started
# Find your api_key here: plot.ly/settings/api

import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')


trace1 = Bar(
    x=['Trial 1', 'Trial 2', 'Trial 3'],
    y=[3, 6, 4],
    name='Control',
    error_y=ErrorY(
        type='data',
        array=[1, 0.5, 1.5],
        visible=True
    )
)
trace2 = Bar(
    x=['Trial 1', 'Trial 2', 'Trial 3'],
    y=[4, 7, 3],
    name='Experimental',
    error_y=ErrorY(
        type='data',
        array=[0.5, 1, 2],
        visible=True
    )
)
data = Data([trace1, trace2])
layout = Layout(
    barmode='group'
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='error-bar-bar', auto_open=False)
